tests/test_edgeql_ddl.py

```diff
@@ -2839,6 +2839,37 @@ async def test_edgeql_ddl_ptr_set_type_validation(self):
                 ALTER TYPE Foo ALTER LINK l SET TYPE Spam USING (SELECT Spam)
             """)
 
+    async def test_edgeql_ddl_ptr_using_dml(self):
```

Comment:
Also need a test for mutations in conversion expressions.

Reply:
You mean casts?

Conversions should be covered by altering the link from multi to single.